### DISCUSSION TITLE
how to use the bicaption in longtblr

### DISCUSSION THREAD

**Discussion by fengzhuo17** [lvjr/tabularray#352] [Q&A]
i can only use the caption but not bicaption in longtblr. i want to use the  bicaption in longtblr. who can help me?

**Comment by lvjr** [comment]
See discussion #235.

**Reply by fengzhuo17** [reply]
I've edited the format using \bicaption, and this will disappear if I use \caption directly.  meanwhile my english title also need the index label.

**Reply by lvjr** [reply]
You can't use `caption` or `bicaption` package to configure captions in `longtblr` tables.

**Reply by fengzhuo17** [reply]
that sound terrible. i think this is not a good  interface for caption in tabularray. i was not well in latex, but if the caption can provide the interface which was same to the using caption, the other package for caption will use easily and meanwhile the tabularray will be well-defined.
